debian/control

```diff
@@ -41,6 +41,8 @@ Build-Depends:
  python-yaml <!nocheck>,
  xsltproc <!nodoc>,
  zlib1g-dev,
+Build-Depends-Indep:
+ libglib2.0-doc,
```

Comment:
I believe you here, but how is this enforced upstream?

Reply:
I’m not quite sure what you mean. How is *what* enforced upstream?

Reply:
I figured this out later. I had assumed this was added to satisfy some build check. But actually the reason to have it is so that the generated docs get cross references to glib's docs.